src/sentry/api/endpoints/project_similar_issues_embeddings_records.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.tasks.backfill_seer_grouping_records import backfill_seer_grouping_records
+
+region_silo_endpoint
+
+
+@region_silo_endpoint
+class ProjectSimilarIssuesEmbeddingsRecords(ProjectEndpoint):
+    owner = ApiOwner.ISSUES
+    publish_status = {
+        "POST": ApiPublishStatus.PRIVATE,
+    }
+
+    def post(self, request: Request, project) -> Response:
+        if not features.has("projects:similarity-embeddings-grouping", project):
+            return Response(status=404)
+        backfill_seer_grouping_records.delay(project)
```

Comment:
could we make it so only superusers can call this endpoint? (can just use `is_active_superuser` for now).  e.g. https://github.com/getsentry/sentry/blob/90d35a1104422d3d40974032aeb673e8f31a77c7/src/sentry/api/bases/sentryapps.py#L492